dmgmodify/main.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func main() {
-	if len(os.Args) != 5 {
+	if len(os.Args) != 4 {
```

Comment:
That didn't work otherwise...

Reply:
Heh. I think in an earlier revision of my work I had a "length" argument here, which I ended up not needing. Clearly I didn't test with this since then.